mri/test_local/test_fourier_adjoint_gpu.py

```diff
@@ -35,11 +35,12 @@ def test_NUFFT_3D(self):
             for platform in self.platforms:
                 _mask = np.random.randint(2, size=(self.N, self.N, self.N))
                 _samples = convert_mask_to_locations(_mask)
-                fourier_op_dir = NonCartesianFFT(samples=_samples,
-                                                 shape=(self.N, self.N,
-                                                        self.N),
-                                                 implementation=platform,
-                                                 n_coils=num_channels)
+                fourier_op_dir = NonCartesianFFT(
+                    samples=_samples,
+                    shape=(self.N, self.N, self.N),
+                    implementation=platform,
+                    n_coils=num_channels
+                )
```

Comment:
not a great idea to have this change here. I understand what you tried to do but I think it's better left unchanged if no significant change is applied.